packages/buidler/contracts/facets/ChallengesFacet.sol

```diff
@@ -386,6 +392,7 @@ contract ChallengesFacet is BaseRelayRecipient {
         } else {
             _challenge.status == Status.Refund;
         }
+        challengeToken().firstMint(_challenge.athlete, _challenge.challengeUrl, _challenge.jsonUrl);
```

Comment:
Nothing should get minted on Status.Refund. Refunds mean the Challenge was never completed

Reply:
Fixed